settings.py

```diff
@@ -677,6 +683,7 @@ class PanicMethod(str):
     enemizer_path: EnemizerPath = EnemizerPath("EnemizerCLI/EnemizerCLI.Core")  # + ".exe" is implied on Windows
     player_files_path: PlayerFilesPath = PlayerFilesPath("Players")
     players: Players = Players(0)
+    allow_quantity: AllowQuantity = False
```

Comment:
```suggestion
    allow_quantity: AllowQuantity | bool = False
```
(could also use typing.Union instead like elsewhere in the file)